rust/ql/test/query-tests/security/CWE-825/AccessAfterLifetime.expected

```diff
@@ -418,12 +420,15 @@ nodes
 | lifetime.rs:730:6:730:7 | r1 | semmle.label | r1 |
 | lifetime.rs:730:11:730:25 | e1.test_match() | semmle.label | e1.test_match() |
 | lifetime.rs:734:12:734:13 | r1 | semmle.label | r1 |
-| lifetime.rs:766:2:766:13 | &val | semmle.label | &val |
-| lifetime.rs:766:2:766:13 | ptr | semmle.label | ptr |
-| lifetime.rs:767:2:767:13 | ptr | semmle.label | ptr |
+| lifetime.rs:766:2:766:11 | &val | semmle.label | &val |
+| lifetime.rs:766:2:766:11 | ptr | semmle.label | ptr |
+| lifetime.rs:766:2:766:11 | ptr | semmle.label | ptr |
```

Comment:
Some of these look like duplicates. Any idea what is going on?

Reply:
I'm a bit surprised by the previous results. When I replaced the macro with the expanded code, I could see two distinct `ptr` results here. So these are legit, they are just squashed together in the macro call location.